homeassistant/components/elkm1/__init__.py

```diff
@@ -85,17 +93,6 @@ def _elk_value(val):
     return (start, end)
 
 
-def _has_all_unique_prefixes(value):
```

Comment:
It's still good to keep this config validation so the user can validate the config without actually importing the config in the config flow.

Reply:
Good call, reverted and pushed